packages/web/stores/profile.ts

```diff
@@ -8,13 +8,17 @@ import {
   toJS,
 } from "mobx";
 
-type Avatar = "wosmongton" | "ammelia";
+type Avatar = "wosmongton" | "ammelia" | "stargaze-pfp";
 
 export class ProfileStore {
   @observable
   private _currentAvatar: Avatar;
+  @observable
+  private _stargazeAvatarUri: string | undefined;
 
   private readonly _avatarStorageKey = "profile_store_current_avatar";
+  private readonly _stargazeAvatarUriStorageKey =
```

Comment:
Can we add their stargaze bech 32 address to this key? That way, when the account is switched it doesn't show an NFT for a different account.